sample-node-config/gcp/scripts/configure-testnet.sh

```diff
@@ -10,17 +10,34 @@ ssh-keyscan github.com >> ~/.ssh/known_hosts
 # download gpg key signing testnet dump
 curl https://api.github.com/users/abailly-iohk/gpg_keys | jq -r '.[] | .raw_key' | gpg --import
 
-# download & verify testnet archive
-curl -o testnet.tar.gz https://storage.googleapis.com/cardano-testnet/testnet.tar.gz
-curl -o testnet.tar.gz.asc https://storage.googleapis.com/cardano-testnet/testnet.tar.gz.asc
-gpg --verify testnet.tar.gz.asc
-
-tar xzf testnet.tar.gz
-
 # get cardano network configuration
 git clone https://github.com/input-output-hk/cardano-configurations
 
-export NETWORK_MAGIC=$(jq .networkMagic cardano-configurations/network/testnet/genesis/shelley.json)
+export NETWORK_MAGIC=$(jq .networkMagic cardano-configurations/network/preview/genesis/shelley.json)
+
+# this is manually hardcoded from https://github.com/input-output-hk/hydra-poc/releases/tag/0.7.0
+# perhaps there would be a way to look those up in the Chain?
+export HYDRA_SCRIPTS_TX_ID=bde2ca1f404200e78202ec37979174df9941e96fd35c05b3680d79465853a246
+
+ln -s cardano-configurations/network/preview devnet
 
+# Mithril stuff
+docker pull ghcr.io/input-output-hk/mithril-client:latest
+SNAPSHOT=$(curl -s https://aggregator.api.mithril.network/aggregator/snapshots | jq -r .[0].digest)
+
+mithril_client () {
+  docker run --rm -ti -e NETWORK=testnet -v $(pwd):/data -e AGGREGATOR_ENDPOINT=https://aggregator.api.mithril.network/aggregator -w /data -u $(id -u) ghcr.io/input-output-hk/mithril-client:latest $@
+}
+
+echo "Restoring snapshot $SNAPSHOT"
+mithril_client show $SNAPSHOT
+mithril_client download $SNAPSHOT
+mithril_client restore $SNAPSHOT
+
+# TODO: set env variable containing path to restored DB
```

Comment:
Is this TODO still relevant?

Reply:
Yes, it's todo with mithril stuff. Will adapt the script with the DB path